page_objects/driverfactory.py

```diff
@@ -94,6 +94,9 @@ def run_browserstack(self, os_name, os_version, browser, browser_version,
         if remote_build_name is not None:
             desired_capabilities = self.remote_build_name(desired_capabilities, remote_build_name)
 
+        #Disable screenshots
+        desired_capabilities['browserstack.debug'] = "false"
```

Comment:
Does `browserstack.debug` enable screenshot by default?

Reply:
The default for browserstack.debug is false i.e if we don't ask it to take screenshots (by setting 'browserstack.debug' to 'true') it won't. However, when I searched on disabling screenshots on browserstack, I found this option and the default to be false, but still the screenshots are getting generated (later I realized that the screenshots are taken by our code). I am explicitly setting it to 'false' so that people know that there is an option like this without going through browserstack documentation. I should not have actually hardcoded this to false, will make it configurable, with the same config parameter I used in Base_Page.py.

Reply:
Do we need to do this for Mobile screenshots too?

Reply:
Thanks @indiranell. The issue was primarily to reduce build time which I could observe can be done by disabling screenshots for tests running on chrome. I do not know if we have the same issue with Mobile. I would suggest to just keep it this way and have this for Mobile if we see the same issues.